instrumentation/net/http/otelhttp/transport.go

```diff
@@ -109,21 +140,64 @@ func (t *Transport) RoundTrip(r *http.Request) (*http.Response, error) {
 		ctx = httptrace.WithClientTrace(ctx, t.clientTrace(ctx))
 	}
 
+	readRecordFunc := func(int64) {}
+	if t.readEvent {
```

Comment:
`readEvent` and `readRecordFunc` map to "read" in the span .. but it's the client's Request object, so it's actually a "write" event - right? because it's being written to the server?

"read" events should be recorded from the bytes consumed by the http.Response.Body, right?